content/blog/2024-05-16.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Version 2.0.0 released!"
+date: 2024-05-15T11:00:00+02:00
+description: "Version 2.0.0 released!"
+draft: false
+---
+
+## It's time to celebrate the 2.0.0 release of Kaoto
+
+With this 2.0.0 release we finished implementing the foundational functionalities for an [Apache Camel](https://camel.apache.org/) visual editor. Kaoto will guide you with the creation of new and the editing of existing [Apache Camel](https://camel.apache.org/) integrations. Together with the other extensions of the [Camel Extension Pack](https://marketplace.visualstudio.com/items?itemName=redhat.apache-camel-extension-pack) it offers functionality from the creation over the local execution with the [Camel CLI](https://camel.apache.org/manual/camel-jbang.html) to even the deployment to your cluster.
```

Comment:
Camel Extension Pack --> Extension Pack for Apache Camel

for legal reason we cannot use `Camel XXX` for project which are not at Apache but only `XXX for Apache Camel`

Reply:
fixed